### PR TITLE
Add knowledge base url override

### PR DESCRIPTION
**Pull request by jojobyte**
Separating https://github.com/dashhive/crowdnode.js/pull/39#discussion_r976119340 into its own PR

### CODE REVIEW

File: lib/crowdnode.js [hunk 1]
```diff
@@ -74,23 +77,25 @@ CrowdNode.responses = {
...
+CrowdNode.init = async function ({ baseUrl, insightBaseUrl, knowledgeBaseUrl }) {
   // TODO use API
   // See https://github.com/dashhive/crowdnode.js/issues/3
 
   CrowdNode._baseUrl = baseUrl;
+  CrowdNode._knowledgeBaseUrl = knowledgeBaseUrl || CrowdNode.main.knowledge;
 
   //hotwallet in Mainnet is XjbaGWaGnvEtuQAUoBgDxJWe8ZNv45upG2
   CrowdNode.main.hotwallet = await request({
     // TODO https://app.crowdnode.io/odata/apifundings/HotWallet
```
Comment (coolaj86) [inline lib/crowdnode.js:91]: TODO => TODONE, please

**Review by coolaj86 — APPROVED** [review]
But please remove the TODONE TODOs. :)

File: lib/crowdnode.js (hunk 1 above)
Comment (jojobyte) [reply lib/crowdnode.js:91]: For documentation purposes and per our other conversation, Those TODO's aren't actually resolved, this just allows bypassing a CORS issue in the browser in vite.

**Comment by jojobyte** [comment]
> But please remove the TODONE TODOs. :)

Those TODO's aren't actually resolved, this just allows bypassing a CORS issue in the browser in vite.

**Pull request merged**